f8a_worker/dispatcher/flows/componentApiFlow.yml

```diff
@@ -1,8 +0,0 @@
----
-  flow-definitions:
-    - name: 'componentApiFlow'
-      queue: '{DEPLOYMENT_PREFIX}_api_componentApiFlow_v0'
-      edges:
-        - from:
-          to:
-            - 'BookkeeperTask'
```

Comment:
@pkajaba We have got a dependency on componentApiFlow in the bayesian coreapi server [[1]](https://github.com/fabric8-analytics/fabric8-analytics-server/blob/master/bayesian/api_v1.py#L307) [[2]](https://github.com/fabric8-analytics/fabric8-analytics-server/blob/master/bayesian/utils.py#L81-L92) and that is the only way we can track the component analysis requests to the system. This will enable us to do user actions analytics in the future. So, let's not remove the flow and the task.